.gitattributes

```diff
@@ -0,0 +1,91 @@
+## GITATTRIBUTES FOR WEB PROJECTS
```

Comment:
Git was converting to Windows-style crlf lin endings, when I want these files to have Linux-style lf endings.